### DISCUSSION TITLE
Allow zombified piglin bypass mob griefing rules

### DISCUSSION THREAD

**Discussion by n0k0m3** [PurpurMC/Purpur#435] [Feature Requests]
## Is your feature request related to a problem?
No

## Describe the solution you'd like.
A similar solution to all of these: https://github.com/pl3xgaming/Purpur/blob/ver/1.17/patches/server/0147-Add-mobGriefing-bypass-to-everything-affected.patch, but for zombified piglin

## Describe alternatives you've considered.
None, enabling mobgriefing is out of the question bc I want to keep zombies out of my doors

**Comment by BillyGalbreath** [comment]
Zoglins (zombified hoglins) dont do anything that would require the mob griefing rule to be bypassed. Unless I'm missing something, they do not break any blocks in the game.

**Comment by n0k0m3** [comment]
They break turtle egg, which is a good bait for gold farm

Edit: extra info: enable turtle-egg.bypass-mob-griefing doesn't work

**Comment by n0k0m3** [comment]
Sorry my brain mixed up zoglins (hoglins) and zomb piglins.
Yep need bypass-mob-griefing for zomb piglins

**Comment by BillyGalbreath** [comment]
Looks like Zombified Piglins inherit the BreakDoor ai goal from the Zombie class. So, setting `mobs.zombie.bypass-mob-griefing` will be true for any mob using that goal (including Vindicators, too).

I'll see what I can do about making separate options for each mob
